### PR TITLE
Readme Update

### PR DESCRIPTION
**Pull request by Stupefacient**
doesn't have DP features
probably needs to be completely rewritten

### CODE REVIEW

**Comment by Stupefacient** [comment]
the gif for opedit broke so I added it back. I don't know how to add folder hence why it isn't in .media

File: README.md [hunk 1]
```diff
@@ -1,353 +1,130 @@
...
+- Dev UI toggle in APK setting.
+- GPS Accurecy on the Dev UI
+- Easily view the EON's IP Address.Just look at the sidebar right under wifi singal strength's.
+- Battery has percentage instead of the battery icon.
+- If the model detect's cut in it will draw two different chevron to show the user that it see's both of the car.
+- Dashcam recording button added to the ui. ( it will save video's to the `/data/media/0/video`)
+- Ability to change the SpeedLimit Offset directly from APK. It is based in percentages. For Example, if -1% at 60mph, it will be  approx. 59.4mph, -10% is roughly 54mph etc. (Thank you eFini for the help)
+- Dynamic Follow Button: Now you can change the Dynamic Follow Distance just by tapping the blue button on the bottom right. Now with colorful DF button depending on the profile
+- Smart speed icon
+- e2e button
```
Comment (rav4kumar) [inline README.md:113]: add to: to-do

File: README.md [hunk 2]
```diff
@@ -1,353 +1,130 @@
...
+- Added ability to turn on and off RSA at certain speeds.
+
+### UI Modifications
+- Dev UI toggle in APK setting.
+- GPS Accurecy on the Dev UI
+- Easily view the EON's IP Address.Just look at the sidebar right under wifi singal strength's.
+- Battery has percentage instead of the battery icon.
+- If the model detect's cut in it will draw two different chevron to show the user that it see's both of the car.
+- Dashcam recording button added to the ui. ( it will save video's to the `/data/media/0/video`)
+- Ability to change the SpeedLimit Offset directly from APK. It is based in percentages. For Example, if -1% at 60mph, it will be  approx. 59.4mph, -10% is roughly 54mph etc. (Thank you eFini for the help)
```
Comment (rav4kumar) [inline README.md]: this wont happen. need to be removed

File: README.md [hunk 3]
```diff
@@ -1,353 +1,130 @@
...
+- 2020 Corolla tuning by Spairrow326
+- Added ability to turn on and off RSA at certain speeds.
+
+### UI Modifications
+- Dev UI toggle in APK setting.
+- GPS Accurecy on the Dev UI
+- Easily view the EON's IP Address.Just look at the sidebar right under wifi singal strength's.
+- Battery has percentage instead of the battery icon.
+- If the model detect's cut in it will draw two different chevron to show the user that it see's both of the car.
+- Dashcam recording button added to the ui. ( it will save video's to the `/data/media/0/video`)
```
Comment (rav4kumar) [inline README.md]: dp always recs. `/sdcard/media/dashcam`

File: README.md [hunk 4]
```diff
@@ -1,353 +1,130 @@
...
+This aims to provide a smoother driving experience in stop and go traffic (under 20 mph) by modifying the maximum gas that can be applied based on your current velocity and the relative velocity of the lead car. It'll also of course increase the maximum gas when the lead is accelerating to help you get up to speed quicker than stock. And smoother; this eliminates the jerking you get from stock ArnePilot with comma pedal. It tries to coast if the lead is only moving slowly, it doesn't use maximum gas as soon as the lead inches forward :). When you are above 20 mph, relative velocity and the following distance is taken into consideration.
+- 2020 Corolla tuning by Spairrow326
+- Added ability to turn on and off RSA at certain speeds.
+
+### UI Modifications
+- Dev UI toggle in APK setting.
+- GPS Accurecy on the Dev UI
+- Easily view the EON's IP Address.Just look at the sidebar right under wifi singal strength's.
+- Battery has percentage instead of the battery icon.
+- If the model detect's cut in it will draw two different chevron to show the user that it see's both of the car.
```
Comment (rav4kumar) [inline README.md]: defatuls in 08

File: README.md [hunk 5]
```diff
@@ -1,353 +1,130 @@
...
+- [Dynamic Gas](https://github.com/ShaneSmiskol/ArnePilot/tree/stock_additions-devel#dynamic-gas)
+This aims to provide a smoother driving experience in stop and go traffic (under 20 mph) by modifying the maximum gas that can be applied based on your current velocity and the relative velocity of the lead car. It'll also of course increase the maximum gas when the lead is accelerating to help you get up to speed quicker than stock. And smoother; this eliminates the jerking you get from stock ArnePilot with comma pedal. It tries to coast if the lead is only moving slowly, it doesn't use maximum gas as soon as the lead inches forward :). When you are above 20 mph, relative velocity and the following distance is taken into consideration.
+- 2020 Corolla tuning by Spairrow326
+- Added ability to turn on and off RSA at certain speeds.
+
+### UI Modifications
+- Dev UI toggle in APK setting.
+- GPS Accurecy on the Dev UI
```
Comment (rav4kumar) [inline README.md]: GPS Accurecy on `sidebar`

File: README.md [hunk 6]
```diff
@@ -1,353 +1,130 @@
...
+- Speed sign reading
+- Stock Toyota ldw steering assist
+- Cruise set speed available down to 7 kph
+- Virtual lane lines and Lane center. This feature is for European roads and is recommended for used in Europe only.
+
+### opEdit features
+all OpEdit features can be manged by running the command `python /data/openpilot/op_edit.py`
+- Ability to ruduce or Increase curvature Factor. It will also works with eco and sport mode. If using eco mode then it will start breaking early (350 m before) if using sport mode it will slow down little late (150 m).
+- Live speedlimit_offset in `op_tune.py`
+- [Dynamic distance profiles](https://github.com/ShaneSmiskol/ArnePilot/tree/stock_additions-devel#dynamic-follow-3-profiles) from Shane (In other word three different dynamic profiles: (`close`, `normal`, `far`, `auto`). Profile can be adjusted from either op_edit.py or using the on screen buttons(can take up to 4 sec to for new profile to be adjusted).
```
Comment (rav4kumar) [inline README.md:95]: `traffic, `normal`, `long?`, `auto`

File: README.md [hunk 7]
```diff
@@ -1,353 +1,130 @@
...
+- RAV4 2017-19
+- Corolla 2019-20
+- Prius 2017-2021
+- RX hyrid 2017
+- CT 2018
+- Chevrolet Volt 2017
+- Subaru Crosstrek Limited 2019 with 0.8 it will use Eyesight for radar. 
+
+### Todo
+- [ ] add Trafficd
```
Comment (rav4kumar) [inline README.md]: wont happen

File: README.md [hunk 8]
```diff
@@ -1,353 +1,130 @@
...
+- [ ] improve tunning
```
Comment (rav4kumar) [inline README.md]: i guess we can say live indi op_edit? so its done

File: README.md [hunk 9]
```diff
@@ -1,353 +1,130 @@
...
+- [ ] add Dev UI
```
Comment (rav4kumar) [inline README.md]: done

File: README.md [hunk 10]
```diff
@@ -1,353 +1,130 @@
...
+- [ ] add OSM and Speed offset apk toggles
```
Comment (rav4kumar) [inline README.md]: wont happen

File: README.md [hunk 11]
```diff
@@ -1,353 +1,130 @@
...
+- [ ] bring back feature op_edit from 0.7
+- [ ] Dynamic distance profiles
```
Comment (rav4kumar) [inline README.md:59]: df is already there

File: README.md [hunk 12]
```diff
@@ -1,353 +1,130 @@
...
+- [ ] Darkmode and community logo
```
Comment (rav4kumar) [inline README.md]: darkmode wont happen.

File: README.md [hunk 13]
```diff
@@ -1,353 +1,130 @@
...
+    - by curvature (mapd),
+    - by mapped sign(stop, yield, roundabouts, bump, hump, traffic light, speed sign, road attribute)
+- No disengage for gas, only longitudinal disengage for brake, tire slip or cancel
+- Only disengage on main off and on brake at low speed
+- Smooth longitudinal controller also at low speeds
+- No disengage for seat belt remove and door opened. Practical for when stopping and then someone opens a door so that the car does not drive into the lead
+- No fingerprint compatibility problems. A completely different way to combine and split Fingerprints so that they always work I.e. comma is not supporting rav4h 2019 because of this Fingerprint method. Mine is better
+- Custom events and capnp structure so that comma is happy with the drives from my fork
+- Forward collision warning actually brakes for you.
+- We also have enabled commas e2e model which will only work between 11 MPH to 29 MPH. Commas e2e model helps slows down for traffic light, stop sign, etc. e2e, traffic model and mapd all works together to help you stop at the light. All of this can be turned off via `/data/openpilot/op_edit.py`.
```
Comment (rav4kumar) [inline README.md]: this need to be taken off or redirect user if they want to use it. to 07x

File: README.md [hunk 14]
```diff
@@ -1,353 +1,130 @@
...
+- Smart speed (smart speed is essentially speedlimit which eon will not go over unless you have set custom offset) can be overridden by pressing gas above the current smart speed.
+- Hands on wheel sensing to comply with European driving regulations by [alfhern](https://github.com/move-fast)
```
Comment (rav4kumar) [inline README.md:81]: this is must. need to be added

File: README.md [hunk 15]
```diff
@@ -1,353 +1,130 @@
...
+- Control 3 gas profiles with sport eco and normal buttons on car (only on limited car)
```
Comment (rav4kumar) [inline README.md]: via ui.

**Review by rav4kumar — COMMENTED** [review]
looks good for the most part.

File: README.md (hunk 11 above)
Comment (Stupefacient) [reply README.md:59]: we are stilling missing the event states for it

**Pull request merged**